src/xo-app/tasks/index.js

```diff
@@ -62,8 +62,12 @@ const COLUMNS = [
   {
     default: true,
     itemRenderer: (task, userData) => {
-      const pool = find(userData.pools, { id: task.$poolId })
-      return <Link to={`/pools/${pool.id}`}>{pool.name_label}</Link>
+      const pool = userData.pools[task.$poolId]
```

Comment:
You  don't need to use `_.find` in `sortCriteria` either. (But GitHub doesn't let me add a comment there :rage:)

Reply:
ok :)